.travis.yml

```diff
@@ -1,39 +1,12 @@
 language: dart
-sudo: required
 dart:
-  - 1.24.2
+  - stable
+  - dev
+sudo: required
 addons:
-  apt:
-    packages:
-    - ttf-kochi-mincho
-    - ttf-kochi-gothic
-    - ttf-dejavu
-    - ttf-indic-fonts
-    - fonts-tlwg-garuda
-before_install:
-  # Content shell needs this font. Since it has a EULA, we need to manually
-  # install it.
-  #
-  # TODO: remove this and use "sudo: false" when travis-ci/travis-ci#4714 is
-  # fixed.
-  - sudo apt-get update -yq
-  - sudo sh -c "echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections"
-  - sudo apt-get install msttcorefonts -qq
-
-  - mkdir -p bin
-  - export PATH="$PATH:`pwd`/bin/"
-  - ln -s `which chromium-browser` bin/google-chrome
-
-  - wget "http://gsdview.appspot.com/dart-archive/channels/stable/release/latest/dartium/content_shell-linux-x64-release.zip"
-  - unzip content_shell-linux-x64-release.zip
-  - ln -s `pwd`/`echo drt-linux-*`/content_shell bin/content_shell
-
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
-  - sleep 3 # give xvfb some time to start
-script:
-  - pub run dart_dev format --check
-  - pub run dart_dev analyze
-  - pub run dart_dev test
-  - pub run dart_dev coverage --no-html
-  - bash <(curl -s https://codecov.io/bash) -f coverage/coverage.lcov
+  chrome: stable
+dart_task:
+  - test: --platform vm
+  - test: --platform chrome
+  - dartfmt: true
+  - dartanalyzer: example lib test tool
```

Comment:
I've been meaning to ask about `dart_task`, it seems overly prescriptive – how are you supposed to handle scenarios where your test task requires an integration server? How do we run a dart_dev task?

Reply:
I agree @evanweible-wf , it just does what it does and I'm not sure how to extend it. But, boy is it simple. Are you ok with using dart_task or would you rather go back to how it was?
My goal was to run CI under both dart 2 and dart 1

Reply:
Yeah I don't mean to say this needs to be addressed on this PR, I think this is fine for w_flux for the purposes of running on Dart 1 and 2 I'm mainly curious what the Dart team's answer is to my question in a more general sense: what is the recommended pattern for when the out-of-the-box `dart_task`s don't cover a CI use case?